tasks/utils.py

```diff
@@ -280,3 +280,14 @@ def load_release_versions(ctx, target_version):
             # environment when running a subprocess.
             return {str(k):str(v) for k, v in versions[target_version].items()}
     raise Exception("Could not find '{}' version in release.json".format(target_version))
+
+def go111module_envvar(command):
```

Comment:
Not a big fan of the name of this function: it's not clear what it does (it could mean that it sets the env var, for example). Maybe something like `check_ go111module_envvar` or `go111module_envvar_not_set` would be more explicit?